docs/source/gbd2017_models/concept_models/vivarium_conic_lsff/concept_model.rst

```diff
@@ -98,6 +98,224 @@ Interventions
+values). **Therefore, we will conservatively use the results from the Cochrane 
+review, with increased certainty in the results based on the confirmatory 
+results from the supplementary meta-analysis.**
+
+Notably, all of these studies included in the supplementary analysis were 
+conducted among children. Additionally, the study locations included 
+Guatemala, South Africa, Nicaragua, Indonesia, and the Phillipines. Therefore, 
+we concluded that it is **reasonable to assume generalizability of these 
+results to our model populations.**
+
+Regarding effect sizes in young age groups, Sandjaja et al. (2015) reported 
+that population vitamin A fortification improved serum retinol concentrations 
+among infants aged 6-11 months. Therefore, **we assumed that the effect size 
+from the Cochrane review applies to all age groups above six months of age.** 
+
+	Notably, the effect can occur either through the direct consumption of 
+	vitamin A fortified foods or through the consumption of breastmilk from 
+	mothers who consume vitamin A fortified foods (Sandjaja et al. 2015; WHO 
+	Guidelines).
+
+For individuals aged between 0 and six months, we made the following 
+assumptions:
+
+	1. Maternal consumption of vitamin A fortified foods has no effect on 
+	infant vitamin A deficiency birth prevalence. This assumption is supported 
+	by studies performed by Dror and Allen (2018).
+
+	2. Maternal consumption of vitamin A fortified foods has no effect on 
+	infant vitamin A deficiency from 0 to six months of age. This assumption 
+	is largely supported by the vitamin A *supplementation* literature among 
+	these age groups and is reflected in WHO guidelines (WHO Guideline: 
+	Vitamin A Supplementation in Infants 1-5 Months of Age).
+
+Additionally, we made assumptions regarding the response time following the 
+onset of exposure to vitamin A fortification, including:
+
+	1. Individuals will exhibit a response in vitamin A deficiency to vitamin 
+	A fortification between appoximately 2 and 12 months after onset of 
+	exposure to vitamin A fortification. There was sparce data available for 
+	the response time to vitamin A fortification, so we used data on the 
+	response time to vitamin D (another fat-soluble vitamin) supplementation 
+	as a proxy. The literature larely indicated that response to vitamin D 
+	supplementation plateaus between 2 and 12 months (Heaney et al. 2008; 
+	Vieth 1999; Taalat et al. 2016; ADDITIONAL). We assumed that the 
+	distribution of response times follows a lognormal distribution with a 
+	median value of five months, a 0.025 percentile of 2 months, and a 0.975 
+	percentile of 12 months.
+
+ 	2. If an individual was covered by baseline coverage of vitamin A 
+ 	fotification, we assumed that the individual was covered (via breastmilk 
+ 	or direct consumption) for long engough to exhibit a response (at least 12 
+ 	months).
+
+**Vivarium Modeling Strategy**
```

Comment:
I think this section is too broad in scope. The entire "Vitamin A Fortification" section is describing the Vivarium modeling strategy. (This is different from cause models where there is a separate GBD modeling strategy.)

I think it would helpful to break up our strategy into discrete chunks and make a separate section and a separate PR for each of those, similar to what I did for folic acid, e.g.

- coverage data
- effect size
- intervention scale-up
- effect of the intervention on simulants
- research considerations